numpy/linalg/setup.py

```diff
@@ -3,11 +3,17 @@
 
 def configuration(parent_package='', top_path=None):
     from numpy.distutils.misc_util import Configuration
-    from numpy.distutils.system_info import get_info, system_info
+    from numpy.distutils.system_info import (
+            get_info, system_info, lapack_opt_info, blas_opt_info)
     config = Configuration('linalg', parent_package, top_path)
 
     config.add_data_dir('tests')
 
+    # Accelerate is buggy, disallow it. See also numpy/core/setup.py
+    for opt_order in (blas_opt_info.blas_order, lapack_opt_info.lapack_order):
+        if 'accelerate' in opt_order:
+            opt_order.remove('accelerate')
```

Comment:
Same question as above ? Shouldn't we remove it completely and then remove those specific tests ? Or is there some subtle interaction I am missing ?

Reply:
@mattip can you briefly comment on this? Should this be an error, or a warning?

Anyway, I would like to put this in pretty much as is (just a general note for those not at the triage meeting).

Reply:
I think adding a warning here might be too noisy: it would trigger whenever a build is done on macOS. Once this PR is in, we had discussed totally removing Accelerate discovery from the code as a single PR that can be reverted if it breaks downstream packages. Note the code here is NumPy specific and does not affect downstream.